userspace/libsinsp/parsers.cpp

```diff
@@ -1287,6 +1287,11 @@ void sinsp_parser::parse_clone_exit(sinsp_evt *evt)
 
 	if(!(tinfo->m_flags & PPM_CL_CLONE_THREAD))
 	{
+		if(m_inspector->m_is_windows)
+		{
+			tinfo->m_flags |= PPM_CL_IS_MAIN_THREAD;
+		}
+
```

Comment:
Is PPM_CL_IS_MAIN_THREAD an exact inverse of PPM_CL_CLONE_THREAD? When can both or none be set?

Reply:
it is not.

PPM_CL_IS_MAIN_THREAD is required because in Windows the TID of a process' main thread is not equal to the process PID (!). This requires us to "mark" the main thread, so that functions like sinsp_thread_info::get_main_thread can work.